File: PHP/Cap03/c03ex06.php
<?php
// c02ex06.php

$palavra = "COMPUTADOR";

printf("Palavra ........: %s\n", $palavra);
printf("\n");
printf("Lado esquerdo ..: %s\n", substr($palavra, 0, 3));
printf("Centro .........: %s\n", substr($palavra, 3, 4));
printf("Lado direito ...: %s\n", substr($palavra, 7, 3));
printf("Reverso ........: %s\n", strrev($palavra));
printf("Copias .........: %s\n", implode('', array_fill(0, 3, $palavra)));
printf("Nova palavra ...: %s\n", str_replace("A", "O", $palavra));

printf("\n");
printf("Tecle <Enter> para encerrar... ");
trim(fgets(STDIN));

?>

File: RUST/Cap03/c03ex06.rs
// c03ex06.rs

use std::io::{self, Write};

fn main() {
  let palavra = "COMPUTADOR";

  println!("Palavra ........: {}", palavra);
  println!();

  println!("Lado esquerdo .: {}", &palavra[..3]);
  println!("Centro ........: {}", &palavra[3..7]);
  println!("Lado direito ..: {}", &palavra[palavra.len() - 3..]); 
  println!("Reverso .......: {}", palavra.chars().rev().collect::<String>());
  println!("Copias ........: {}", palavra.repeat(3));
  println!("Tira pedaco ...: {}", &palavra[..7]);
  println!("Nova palavra ..: {}{}{}", &palavra[..6], "O", &palavra[7..]);

  println!();
  print!("Tecle <Enter> para encerrar... ");
  io::stdout().flush().expect("Falha ao limpar o buffer");
  let mut _entrada = String::new();
  io::stdin().read_line(&mut _entrada).expect("Falha ao ler a entrada");
}
